distribution/src/main/scripts/fscrawler.bat

```diff
@@ -21,6 +21,7 @@ set JAVA_OPTS=%JAVA_OPTS% -Djava.awt.headless=true
 
 REM Ensure UTF-8 encoding by default (e.g. filenames)
 set JAVA_OPTS=%JAVA_OPTS% -Dfile.encoding=UTF-8
+set JAVA_OPTS=%JAVA_OPTS% -Dsun.jnu.encoding=UTF-8
```

Comment:
This should not be part of this PR as it's coming in another one.

Reply:
Yes I understood. The previous PR was just approved but still open. That's why. I have to create a PR from patch. Either way I will have a clean PR.